### PR TITLE
Feature 1 is completed

### PR DESCRIPTION
**Pull request by Bhanuprakash-1995**
- Feature 1 development is completed
- added user signup
- added login signup

### CODE REVIEW

File: learn-git/README.MD [hunk 1]
```diff
@@ -1,4 +1,9 @@
 ## Git
 This is my first commit from main branch
 This is my second commit
-This is my third commit
\ No newline at end of file
+This is my third commit
+#########
+This is feature-1
```
Comment (singalasreenivasulu55) [inline learn-git/README.MD:6]: Please reomve line5

Comment (Bhanuprakash-1995) [reply learn-git/README.MD:6]: I have updated please review once again

Comment (singalasreenivasulu55) [reply learn-git/README.MD:6]: Looks good and approving.

**Review by singalasreenivasulu55 — APPROVED** [review]
Looks goods to me and approving.

**Pull request merged**